proto/store/setting.proto

```diff
@@ -33,6 +33,27 @@ message WorkspaceProfileSetting {
 
   // The duration for refresh token.
   google.protobuf.Duration refresh_token_duration = 6;
+
+  // The setting of custom announcement
+  Announcement announcement = 7;
+}
+
+message Announcement {
+  // We support three levels of AlertLevel: INFO, WARNING, and ERROR.
+  enum AlertLevel {
+    ALERTLEVEL_INFO = 0;
```

Comment:
> The first value of the enum should be the name of the enum itself followed by the suffix _UNSPECIFIED.

Following https://google.aip.dev/126, we should add a `suffix_UNSPECIFIED` as the first value.